source/slang/slang-emit-spirv.cpp

```diff
@@ -2765,14 +2765,14 @@ struct SPIRVEmitContext
                     if (isQuad)
                     {
                         verifyComputeDerivativeGroupModifiers(this->m_sink, inst->sourceLoc, true, false, numThreadsDecor);
-                        emitOpExecutionMode(getSection(SpvLogicalSectionID::ExecutionModes), nullptr, entryPoint, SpvExecutionModeDerivativeGroupQuadsNV);
-                        emitOpCapability(getSection(SpvLogicalSectionID::Capabilities), nullptr, SpvCapabilityComputeDerivativeGroupQuadsNV);
+                        requireSPIRVExecutionMode(nullptr, getID(ensureInst(entryPoint)), SpvExecutionModeDerivativeGroupQuadsNV);
```

Comment:
Can we just do getID(entryPoint)?

Reply:
I can make it so `getID(IRInst)` calls `getID(ensureInst(entryPoint))`?

Reply:
Just use `getIRInstSpvID`.